### DISCUSSION TITLE
how to modify emqx.conf in emqxBroker.yaml

### DISCUSSION THREAD

**Discussion by falser101** [emqx/emqx-operator#494] [Q&A] [answered]
`emqxTemplate:
  properties:
       acl:
       listener:
       modules:
       plugins:
`
the crd only has this properties. how can i modify the emqx.conf???

**Comment by falser101** [comment]
@Rory-Z

**Comment by Rory-Z** [comment, marked as answer]
You can use `.spec.emqxTemplate.config`
